spec/menu_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'menu'
+
+describe Menu do
+
+  subject(:menu) { Menu.new(dishes) }
+  let(:dishes) do { "Ramen" => 2 }
```

Comment:
Style/MultilineBlockLayout: Block body expression is on the same line as the block start.